src/controller/GameController.java

```diff
@@ -275,13 +283,17 @@ public static void main(String[] args) {
             Rules.league.delayedSwitchAfterGoal = 0;
         }
 
+        InetAddress broadcastAddress = localAddress.getBroadcast() == null ? localAddress.getAddress() : localAddress.getBroadcast();
```

Comment:
can this be made final?
```suggestion
        final InetAddress broadcastAddress = limitedBroadcast ? InetAddress.getByName("255.255.255.255") : localAddress.getBroadcast() == null ? localAddress.getAddress() : localAddress.getBroadcast();
```

Reply:
`InetAdress.getByName` can throw an `UnknownHostException`, so not without moving this and also the Log prints into a common try-catch-block.